yt/frontends/exodus_ii/data_structures.py

```diff
@@ -318,7 +318,7 @@ def _read_coordinates(self):
             else:
                 coords = np.array([coord for coord in
                                    ds.variables["coord"][:]]).transpose().copy()
-            return coords
+            return coords.astype(np.float64)
```

Comment:
Since `astype` copies the data by default, we could avoid an extraneous copy here by replacing the calls to `.copy()` on line 317 and 320 by `.astype(np.float64)`.
If you want to make sure the copy is done, you can use `astype(np.float64, copy=True)` which is the same as the default but makes the intention clear.

Reply:
That's a good point! I'll go ahead and make that change.